### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by vegichan**
Do not have a bug template set up. This proposal will create a default template for anyone who files issues against this GitHub repo.

It will show up here:
https://github.com/nuodb/nuodb-helm-charts/issues/new

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
```
Comment (NikTJ777) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]: Should we have an "external reference" field, that can hold either a JIRA number or a ZenDesk tickert ref?

Comment (vegichan) [reply .github/ISSUE_TEMPLATE/bug_report.md:7]: I dont think that I can add it to the auto-generated list of labels, but we can definitely add it to the end.

something like:?

External Reference
ZenDesk/JIRA numbers if available

**Review by acabrele — APPROVED** [review]

**Review by sivanov-nuodb — APPROVED** [review]
Thanks.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (NikTJ777) [reply .github/ISSUE_TEMPLATE/bug_report.md:7]: Perfect! :)

**Pull request merged**